src/introduction/what's-new-2.0.md

```diff
@@ -0,0 +1,45 @@
+# What's New in 2.0
+
+The release of CosmWasm 2.0 marks a significant milestone in the evolution of the CosmWasm framework, introducing a host of new features, performance improvements, and security enhancements. This version builds upon the solid foundation of previous releases, offering developers more tools and capabilities to create robust, efficient, and secure decentralized applications. Key updates include advanced interoperability through IBC integration, optimized smart contract performance, and comprehensive security upgrades, ensuring that CosmWasm remains at the forefront of smart contract development technologies.
+
+## From 1.5 to 2.0: A Leap Forward
+
+The transition from CosmWasm 1.5 to 2.0 marks a significant leap in the framework's evolution. This upgrade responds to the growing needs of the developer community and the ever-changing landscape of blockchain technology.
+
+## Changelog Highlights
+
+For a detailed list of all changes, improvements, and bug fixes in CosmWasm 2.0, we encourage you to review the official changelog on [GitHub](https://github.com/CosmWasm/cosmwasm/blob/main/CHANGELOG.md). The changelog provides an exhaustive breakdown of the updates, offering insights into the development focus areas and enhancements that have been prioritized in this release.
+
+## Highlights from the changelog include:
+
+- **IBC Integration**: Enhanced support for Inter-Blockchain Communication (IBC), allowing CosmWasm contracts to seamlessly interact with other IBC-enabled blockchains.
+- **Performance Optimizations**: Significant improvements in contract execution speed and efficiency, reducing gas costs and improving the overall user experience.
+- **Security Enhancements**: Introduction of new security protocols and measures to strengthen contract integrity and protect against emerging vulnerabilities.
```

Comment:
The only "Security Enhancement" we did is this: https://github.com/CosmWasm/cosmwasm/pull/1991/files. It's an improved fix for one very specific vulnerability we had, but that's about it. All the other supported versions are also fixed, just not with the improved fix.